phpBB/styles/prosilver/template/forum_fn.js

```diff
@@ -156,7 +156,11 @@ function selectCode(a) {
 		// Safari and Chrome
 		if (s.setBaseAndExtent) {
 			var l = (e.innerText.length > 1) ? e.innerText.length - 1 : 1;
-			s.setBaseAndExtent(e, 0, e, l);
+			try {
+				s.setBaseAndExtent(e, 0, e, l);
+			} catch (nil) {
```

Comment:
Yeah I wasn't a huge fan of `nil` either, but didn't want to nitpick. It's misleading.